### ISSUE TITLE
handleInstallPath throws an error when stateVerification is false

### ISSUE THREAD

**Issue by rockingskier** [slackapi/node-slack-sdk#1506]
I have my app enabled for org-wide installs.
Org apps require `stateVerification` is set to `false`, however, this breaks the `/slack/install` handler.

A `stateStore` is only created if you have `stateVerification=true`: https://github.com/slackapi/node-slack-sdk/blob/main/packages/oauth/src/install-provider.ts#L133

Down the line in `handleInstallPath` it bails out when no `stateStore` exists: https://github.com/slackapi/node-slack-sdk/blob/main/packages/oauth/src/install-provider.ts#L340-L342

I believe this was introduced in #1436.

It's very possible I'm missing something but it was working with `"@slack/bolt": "^3.10.0",`.

#### Packages:

- [ ] `@slack/web-api`
- [ ] `@slack/rtm-api`
- [ ] `@slack/webhooks`
- [x] `@slack/oauth`
- [ ] `@slack/socket-mode`
- [ ] I don't know

### Reproducible in:
Definitely 2.5.3
Probably from 2.5.0 (this was the first release after #1436)

```bash
"slack/bolt": "^3.11.3",
"slack/logger": "^3.0.0",
"slack/web-api": "^6.7.2",
```
#### Node.js runtime version

`v16.14.0`

#### OS info

```
ProductName:	macOS
ProductVersion:	11.6.7
```

#### Steps to reproduce:

1. Enable the app for org installation
2. Pass `stateVerification` as `false`
```
new ExpressReceiver({
    // config
    installerOptions: {
      stateVerification: false,
    },
    // more config
})
```
3. Run the app
4. Visit `/slack/install`

### Expected result:

A nice "Add to Slack" button

### Actual result:
```
Error: StateStore is not properly configured
    at new GenerateInstallUrlError (/MY_APP/node_modules/@slack/oauth/src/errors.ts:22:1)
    at InstallProvider.<anonymous> (/MY_APP/node_modules/@slack/oauth/src/install-provider.ts:341:13)
    at step (/MY_APP/node_modules/@slack/oauth/dist/install-provider.js:44:23)
    at Object.next (/MY_APP/node_modules/@slack/oauth/dist/install-provider.js:25:53)
    at /MY_APP/node_modules/@slack/oauth/dist/install-provider.js:19:71
    at new Promise (<anonymous>)
    at __awaiter (/MY_APP/node_modules/@slack/oauth/dist/install-provider.js:15:12)
    at InstallProvider.handleInstallPath (/MY_APP/node_modules/@slack/oauth/dist/install-provider.js:295:16)
    at /MY_APP/node_modules/@slack/bolt/src/receivers/ExpressReceiver.ts:283:29
    at Layer.handle [as handle_request] (/MY_APP/node_modules/@slack/bolt/node_modules/express/lib/router/layer.js:95:5)
```

### Requirements

Please read the [Contributing guidelines](https://github.com/slackapi/node-slack-sdk/blob/main/.github/contributing.md) and [Code of Conduct](https://slackhq.github.io/code-of-conduct) before creating this issue or pull request. By submitting, you are agreeing to those rules.

**Comment by seratch** [comment]
@rockingskier Thanks for taking the time to report this. I agree that this should be improved in future releases.

**Comment by rockingskier** [comment]
Happy to help 👍

**Comment by seratch** [comment]
Fixed by #1507
